osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -28,7 +53,21 @@ protected override double StrainValueOf(DifficultyHitObject current)
             var osuCurrent = (OsuDifficultyHitObject)current;
 
             double result = 0;
-
+            if (osuCurrent.JumpDistance < 150)
+            {
+            double sectionvelocity = osuCurrent.JumpDistance/osuCurrent.StrainTime;
+            string tempstor = Path.GetTempPath();
+            if (sectionvelocity < streamaimconst) 
+                { 
+                    string comma = "" ;
```

Comment:
comma should be "," right?

Reply:
That returns unexpected pp results, will look into that tomorrow morning.

Reply:
This happens because of different [cultures](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo?view=netcore-3.1) on different systems. Yours probably has `,` as digit separator

Reply:
Fixed in #8 .